.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
+# copied from https://black.readthedocs.io/en/stable/compatible_configs.html#configuration
+multi_line_output = 3
+include_trailing_comma = True
+force_grid_wrap = 0
+use_parentheses = True
+ensure_newline_before_comments = True
+line_length = 88
```

Comment:
Why do we need black config? If this seems to be default value there is maybe no need to add it?

Reply:
please see https://github.com/SeldonIO/seldon-core/pull/2927. I realize there is a better way to configure compatibility between black and isort now. Will remove this after https://github.com/SeldonIO/seldon-core/pull/2927 is merged.

Reply:
Can we separate the two and first get in the `bins`? 

For the `isort`/`black` maybe we could start first with issue describing what is the change and what it adds?

Reply:
I thought it would be easier to get the isort/black thing first. The addition of isort is just to sort imports nothing changed to the logic of the code.

After https://github.com/SeldonIO/seldon-core/pull/2927 is merged, I will rebase and update this pull request. so the two are separated

Reply:
Ok, fair enough. Will look at #2927 .

Reply:
If you really prefer to get this PR done first. I'd be fine with it too. please let me know.

Reply:
No, it's okay. If this is only sorting imports it's fine. Asked a question on #2927 and triggered integration tests there.